.node-version

```diff
@@ -0,0 +1 @@
+12.16.3
```

Comment:
Do we want to put a caret in front of that to allow some minor+patch [flex](https://nodesource.com/blog/semver-tilde-and-caret/)? `^12.16.3`